src/Terminal/commands/help.ts

```diff
@@ -1,20 +1,104 @@
-import { Terminal } from "../../Terminal";
+import { Player } from "@player";
+import type { AutocompleteData } from "@nsdefs";
 import { TerminalHelpText, HelpTexts } from "../HelpText";
+import { Terminal } from "../../Terminal";
+import { compile } from "../../NetscriptJSEvaluator";
+import { wrapUserNode } from "../../Netscript/NetscriptHelpers";
+import { enums } from "../../NetscriptFunctions";
+import { GetAllServers } from "../../Server/AllServers";
+import { hasScriptExtension, resolveScriptFilePath, ScriptFilePath } from "../../Paths/ScriptFilePath";
+import { Flags } from "../../NetscriptFunctions/Flags";
+import { hasTextExtension } from "../../Paths/TextFilePath";
 
 export function help(args: (string | number | boolean)[]): void {
   if (args.length !== 0 && args.length !== 1) {
     Terminal.error("Incorrect usage of help command. Usage: help");
     return;
   }
   if (args.length === 0) {
-    TerminalHelpText.forEach((line) => Terminal.print(line));
+    const flushedOut = TerminalHelpText.join("\n");
+    Terminal.print(flushedOut);
   } else {
     const cmd = args[0] + "";
     const txt = HelpTexts[cmd];
+
     if (txt == null) {
-      Terminal.error("No help topics match '" + cmd + "'");
+      // Here is where flow lands if we have a player-implemented command
+
+      // Input sanitization
+      const cmdCopy = String(cmd).replace(/^[/.]+/, "") as ScriptFilePath;
```

Comment:
Why this? Does help not work for scripts with relative directories?

This doesn't feel like the right way to be checking for a script. I think Terminal.getFilepath is what you ought to be using here.